src/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ if (WIN32)
                        ${DLL_LOCATION} ${UNIT_TEST_DIR})
 else(WIN32)
     find_package(LibXml2 2.9 REQUIRED)
-    find_package(CURL 7.31 REQUIRED)
+    find_package(CURL 7.29 REQUIRED)
```

Comment:
I forked "ds3_c_sdk" repo it already: https://github.com/archive-engines/ds3_c_sdk
I have a question: are you going to support painless build and installation of DS3 C SDK on RHEL7 based systems?